templates/registration/password_change_form.html

```diff
@@ -2,6 +2,10 @@
 
 {% block body %}
     <form action="" method="post" class="form-area">
+        {% if request.session.password_pwned %}
+            <h4>{{ _('We found your password in a <a href="https://haveibeenpwned.com/Passwords">database of compromised passwords</a>. 
```

Comment:
Can you produce a screenshot to show this working correctly?

Reply:
![image](https://user-images.githubusercontent.com/31937503/79544831-9fd3e780-805d-11ea-9018-b549482bf354.png)